src/test/java/PlaySongTests.java

```diff
@@ -0,0 +1,59 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class PlaySongTests extends BaseTest {
+
+    @Test
+    public void playSongs() throws InterruptedException {
+
+        String expectedMessage = null;
+
+        //Standard procedures
+        navigateToPage();
+        enterEmail("demo@testpro.io");
+        enterPassword("te$t$tudent");
+        submit();
+        Thread.sleep(3000);
+        playNextSongBtn();
+        playSongBtn();
+        findSoundBars();
```

Comment:
**Use in test methods, the following logical split   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

    
For example

![18](https://github.com/Test-Pro-Bootcamp/A59-QA-Automation-Framework/assets/104761482/28f62abe-0702-4bea-876e-09f82e8cf9c9)